plugins/app/src/extensions/AppRoot.tsx

```diff
@@ -141,13 +143,14 @@ function SignInPageWrapper({
   const [identityApi, setIdentityApi] = useState<IdentityApi>();
   const configApi = useApi(configApiRef);
   const basePath = getBasePath(configApi);
+  const signOutTargetUrl = getSignOutTargetUrl(configApi);
```

Comment:
Probably a good idea to validate the configuration, e.g., strip out query parameters, don't accept external address, and ensure it's a valid route and etc

Reply:
This is covered in my long-form description above. External URLs, query parameters, and unverifiable routes are necessary to support this use case, unfortunately.

Reply:
Got it, that makes sense. Thanks for sharing the use case requirements 🙂